docs/data-ai/ai/create-dataset.md

```diff
@@ -14,320 +14,310 @@ aliases:
-                           y_max_normalized=box["yMaxNormalized"]
-                       )
+1. Run the following command to create a dataset, replacing the `<org-id>` and `<name>` placeholders with your organization ID and a unique name for the dataset:
 
-               file_ids.append(binary_id)
+   ```sh {class="command-line" data-prompt="$"}
+   viam dataset create --org-id=<org-id> --name=<name>
+   ```
 
-       await data_client.add_binary_data_to_dataset_by_ids(
-           binary_ids=file_ids,
-           dataset_id=dataset_id
-       )
-       print("Added files to dataset.")
-       print("https://app.viam.com/data/datasets?id=" + dataset_id)
+{{% /tab %}}
+{{< /tabs >}}
 
-       viam_client.close()
+## Capture images
 
-   if __name__ == '__main__':
-       asyncio.run(main())
-   ```
+{{< tabs >}}
+{{% tab name="One image" %}}
 
-1. Run the script to upload the images and their metadata into a dataset in Viam app providing the following input:
+You can add images to a dataset directly from a camera or vision component feed in the **CONTROL** or **CONFIGURATION** tabs of the Viam app.
 
-   ```sh {class="command-line" data-prompt="$" }
-   python upload_data.py -org-id <ORG-ID> -api-key <API-KEY> \
-      -api-key-id <API-KEY-ID> -machine-part-id <MACHINE-PART-ID> \
-      -location-id <LOCATION-ID> -dataset-name <NAME>
-   ```
+To add an image directly to a dataset from a visual feed, complete the following steps:
 
-1. Continue to [Train a tflite machine learning model](/data-ai/ai/train-tflite/).
+1. Open the **TEST** panel of any camera or vision service component to view a feed of images from the camera.
+1. Click the button marked with the camera icon to save the currently displayed image to a dataset:
+   {{< imgproc src="/components/camera/add_image_to_dataset_button.png" alt="A button marked with the outline of a camera, emphasized in red" resize="800x" style="width:500px" class="imgzoom" >}}
+1. Select an existing dataset.
+1. Click **Add** to add the image to the selected dataset.
+1. When you see a success notification that reads "Saved image to dataset", you have successfully added the image to the dataset.
 
-{{% /expand%}}
+To view images added to your dataset, go to the **DATA** page's [**DATASETS** tab](https://app.viam.com/data/datasets) in the Viam app and select your dataset.
 
-## Prerequisites
+{{% /tab %}}
+{{% tab name="Many images" %}}
 
-{{< expand "At least 10 captured images of what you want your machine to recognize." >}}
+To capture a large number of images for training an ML model, [Capture and sync image data](/data-ai/capture-data/capture-sync/) using the data management service with your camera.
 
-[Capture and sync image data](/data-ai/capture-data/capture-sync/) using the data management service.
+Viam stores the images saved by capture and sync on the [**DATA** page](https://app.viam.com/data/), but does not add the images to a dataset. Visit the **DATA** page to add images to a dataset.
```

Comment:
Maybe this since it's faster that way? 
```suggestion
Viam stores the images saved by capture and sync on the [**DATA** page](https://app.viam.com/data/), but does not add the images to a dataset.
We recommend you tag the images first and then use the CLI to [add the tagged images to a dataset](/data-ai/ai/create-dataset/#add-tagged-images-to-a-dataset).
```